packages/shared/src/styles/constants.scss

```diff
@@ -224,6 +224,32 @@ $gradient-webmoney: linear-gradient(90deg, #1a77ac 0%, #0068a3 100%);
 $gradient-virtual: linear-gradient(274.25deg, #333333 9.01%, #5c5b5b 103.31%);
 $gradient-virtual-swap-free: linear-gradient(58.51deg, #061828 28.06%, #1a3c60 93.51%);
 
+/* Appstore Banner grandient background */
+$mt5-migration-banner-light-desktop: linear-gradient(
+    90deg,
+    #fff 0%,
+    rgba(255, 255, 255, 0.65) 64.58%,
+    rgba(255, 255, 255, 0) 100%
+);
+$mt5-migration-banner-dark-desktop: linear-gradient(
+    90deg,
+    #0e0e0e 0%,
```

Comment:
Can u use existing constants for color. Like `$COLOR_BLACK_3` instead of `#0e0e0e`